Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -19,25 +20,29 @@ class TransactionHistoryViewController: UIViewController {
         return refreshControl
     }()
     
-    @IBOutlet weak var syncLabel: UILabel!
     
+    @IBOutlet weak var syncLabel: UILabel!
     @IBOutlet var tableView: UITableView!
     @IBOutlet var btnFilter: DropMenuButton!
```

Comment:
Also, consider renaming `btnFilter`, it really isn't a button despite being subclassed from `UIButton`. `transactionFilterDropDown` sounds better.

Reply:
fixed in 28f46d1